nodejs/week1/homework/src/backend/app.js

```diff
@@ -0,0 +1,92 @@
+const express = require("express");
+const app = express();
```

Comment:
Nitpick(readability):
Prefer to put all imports in one place, then global variables and finally blocks for each route. Something like this:
```js
// All imports 
const express = require("express");
const meals = require("./data/meals");

// All global variables 
const app = express(); 

// One block per route 
app.get("/", async (request, response) => { ... }); 
app.get("/meals", async (request, response) => { ... });
```

Further, try to be careful of where you put the whitelines, compare:
```js

// Comment about the task

app.get('/meals', async (request, response) => {

  // Some code

});


app.get('/meals', async (request, response) => {
  // Some code

});
```

with:
```js
// Comment about the task
app.get('/meals', async (request, response) => {
  // Some code
});

// Comment about the task
app.get('/meals', async (request, response) => {
  // Some code
});
```

Being consistent with where you put empty lines makes the code easier to read as it flows.